js/controller.js

```diff
@@ -275,6 +275,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onEmbedCodeChanged: function(event, embedCode, options) {
       this.state.videoQualityOptions.availableBitrates = null;
+      this.state.closedCaptionOptions.availableLanguages = null;
```

Comment:
@ksunika Is it safe to wipe the discoveryData here? If we're watching a series of discovery videos, would that risk upsetting our place in them as each video loaded?